config/examples/kontron_vx3060_s2.config

```diff
@@ -1,8 +1,8 @@
 ARCH=x86_64
 TARGET=kontron_vx3060_s2
 WOLFBOOT_SMALL_STACK=0
-SIGN=ECC384
-HASH=SHA384
+SIGN=ECC256
```

Comment:
We should default to ECC384 for the examples, right?

Reply:
To have ECC384 working right now two signing keys must be added to the keystore, one for authenticating the images and one for authenticating the policy for the sealed secret. I think it's better to postpone an example with different keys from TPM and images in a different PR.